api/build.gradle

```diff
@@ -4,7 +4,9 @@ import org.gradle.api.tasks.JavaExec
 import org.pmiops.workbench.tooling.GenerateAPIListingTask
 import org.pmiops.workbench.tooling.IncrementalHotSwapTask
 
-def swaggerTemplateDir = 'src/main/resources'
+// This path is the Java sources root for finding any overridden Swagger-codegen
+// templates we may be using.
+def swaggerTemplateDir = 'src/main/resources/Java'
```

Comment:
Good idea – I renamed to "swaggercodegen". I think at some point I'd thought the exact name of the template directory affected whether Swagger would pick it up, but I confirmed otherwise.